noir/test_programs/execution_success/double_verify_proof/src/main.nr

```diff
@@ -2,31 +2,34 @@ use dep::std;
 
 fn main(
     verification_key: [Field; 114],
-    proof: [Field; 94],
+    // This is the proof without public inputs attached.
+    // 
+    // This means: the size of this does not change with the number of public inputs.
+    proof: [Field; 93],
```

Comment:
This was quite confusing before as it would increase whenever we had a circuit with additional public inputs